docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>"The setup function takes as arguments a list of outputs and a list of inputs. The first dimension is the index of the output/input, and the second dimension is the sample index."</p><p> </p><p>BTW, is it a list or an np.array?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>It's a list. I wasn't able force numba to get np.array.</p>

Reply:
<p>The setup function calculates the shapes of the outputs. It takes two arguments: a list of output shapes, which it fills, and a list of input shapes, which contains the input shape.</p><p> </p><p>BTW - when is this function invoked? Does it also get compiled? Are there any constraints on what it can do? I think that we should at least mention it here (perhaps provide some reference to the relevant part of Numba documentation).</p>